plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/index.tsx

```diff
@@ -25,6 +26,11 @@ const isBlockifiedAddToCart = getSettingWithCoercion(
 	isBoolean
 );
 
+export interface Attributes {
+	className?: string;
+	isDescendentOfSingleProductBlock: boolean;
+}
+
```

Comment:
Nitpick: what about moving this to `plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/types.ts`? The file is already there so seems like this interface would belong there 🙌

Reply:
Oh right, good catch! Done in f128d8af177ecc75d857eb168f10608540e07e13.